x/upgrade/client/cli/tx.go

```diff
@@ -94,7 +94,7 @@ func NewCmdSubmitCancelUpgradeProposal() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "cancel-software-upgrade [flags]",
 		Args:  cobra.ExactArgs(0),
-		Short: "Submit a software upgrade proposal",
+		Short: "Submit a cancel previous software upgrade proposal",
```

Comment:
```suggestion
		Short: "Cancel the current software upgrade proposal",
```

Reply:
Haha yes of course! I was thinking about it, but I thought it was better to keep the same format, if you think that's better of course!